templates/keepalived.default

```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+# Options to pass to keepalived
```

Comment:
This comment is not really helpful as it is a repetition of the comment on line 4.

Reply:
It comes from the file installed by Debian!
I think they anticipated other variables

Reply:
Ok, we can keep it then.